cmd/experimental/kjobctl/Makefile

```
@@ -17,6 +17,14 @@ CGO_ENABLED ?= 0
 GO_CMD ?= go
 GO_FMT ?= gofmt
 
+GIT_TAG ?= $(shell git describe --tags --dirty --always)
+
+DOCKER_BUILDX_CMD ?= docker buildx
+IMAGE_BUILD_CMD ?= $(DOCKER_BUILDX_CMD) build
+STAGING_IMAGE_REGISTRY ?= us-central1-docker.pkg.dev/k8s-staging-images
+IMAGE_REGISTRY ?= $(STAGING_IMAGE_REGISTRY)/kjob
```

```suggestion
IMAGE_PROJECT ?= kjob
IMAGE_REGISTRY ?= $(STAGING_IMAGE_REGISTRY)/$(IMAGE_PROJECT)
```

This way we can temporary overwrite it in `cloudbuild.yaml` of kueue.